engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/subscriptions-helper.service.js

```diff
@@ -49,7 +49,7 @@ angular.module('Bastion.subscriptions').service('SubscriptionsHelper',
 
             selected = [];
             angular.forEach(table.getSelected(), function (subscription) {
-                selected.push({"id": subscription.id});
+                selected.push({"id": subscription.cp_id});
```

Comment:
what logic was this exposing that caused it to remove all subs?  Looking at the controller i don't see why this fixes it (or that this was the problem).